### PR TITLE
Cml/removeheaders

### PR DESCRIPTION
**Pull request by cmluciano**
this feature will be implemented once https://github.com/istio/istio/pull/8183 is merged

### CODE REVIEW

**Comment by cmluciano** [comment]
@frankbu the implementation is now merged. Please review and approve this docs update when you get a chance

File: networking/v1alpha3/virtual_service.proto [hunk 1]
```diff
@@ -390,7 +390,6 @@ message HTTPRoute {
   map<string, string> append_headers = 11;
 
   // Http headers to remove before returning the response to the caller
```
Comment (frankbu) [inline networking/v1alpha3/virtual_service.proto]: Need to add a period at the end of the sentence.

**Pull request merged**